l3build-zip.lua

```diff
@@ -22,18 +22,19 @@ for those people who are interested.
 
 --]]
 
-local concat = table.concat
-local open = io.open
-local osdate = os.date
-local pack = string.pack
+local concat  = table.concat
+local open    = io.open
+local os_date = os.date
+local os_time = os.time
+local pack    = string.pack
 local setmetatable = setmetatable
-local iotype = io.type
+local iotype  = io.type
```

Comment:
Not really seeing the need for this but at the same time I have no strong objection